linkerd/app/integration/src/tests/telemetry.rs

```diff
@@ -1191,6 +1191,8 @@ mod transport {
         test_tcp_connect(TcpFixture::outbound()).await;
     }
 
+    // XXX This test is flakey when running coverage tests.
+    #[cfg_attr(not(feature = "flaky_tests"), ignore)]
```

Comment:
i believe the `flakey_tests` feature is disabled when running the normal CI test job, as well, so these tests will now never run on CI. should we have a separate cfg to disable them only under coverage?

Reply:
I'm pretty sure these tests can actually be flakey in ci too :/ I don't have a good catalog of flakey test failures, but they show up in normal CI runs from time to time.  So, I'd really _love_ to figure out how to test this stuff better in general...

I guess we could add a `very_flaky_test` feature for now, though.